src/pages/IssueToken/components/IssueTokenForm.tsx

```diff
@@ -0,0 +1,176 @@
+import classNames from 'classnames';
+import { Button, MxLink } from 'components';
+import { DataTestIdsEnum } from 'localConstants';
+
+import { routeNames } from 'routes';
+import { useIssueToken } from '../hooks';
+import { IssueTokenFieldsEnum } from '../types';
+
+export const IssueTokenForm = () => {
+  const { formik } = useIssueToken();
+
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      noValidate
```

Comment:
Does this property exist on the default HTML5 `form`?

Reply:
https://www.w3schools.com/tags/att_form_novalidate.asp

Reply:
Cool. 👍